libs/partners/openai/tests/integration_tests/chat_models/test_base_standard.py

```diff
@@ -20,3 +23,69 @@ def chat_model_params(self) -> dict:
     @property
     def supports_image_inputs(self) -> bool:
         return True
+
+    def test_image_token_counting_jpeg(self, model: BaseChatModel) -> None:
```

Comment:
(nit) typically standard tests test methods that are implemented on `ChatModelIntegrationTests`. If we think other providers will support estimating image token counts then I think it makes sense to leave here as a "todo" to eventually migrate to langchain-standard-tests. Otherwise could make more sense in tests/integration_tests/chat_models/test_base.py.

Reply:
not sure any other models have oss tokenizers atm so seems premature to add to standard tests, but down to leave a todo